tests/system/action/meeting/test_import.py

```diff
@@ -1270,6 +1273,34 @@ def test_check_hit_limit_of_users(self) -> None:
             == response.json["message"]
         )
 
+    def test_merge_users(self) -> None:
+        self.set_models(
+            {
+                "user/14": {
+                    "username": "username_test",
+                    "first_name": None,
+                    "last_name": None,
+                    "email": "test@example.de",
+                }
+            }
+        )
+        request_data = self.create_request_data(
+            {
+                "user": {
+                    "12": {
```

Comment:
Put a second new user into the request data, to demonstrate the different handling.

Reply:
Added a second "new" user in the payload.